src/doc/rustdoc/src/command-line-arguments.md

```diff
@@ -0,0 +1,358 @@
+# Command-line arguments
+
+Here's the list of arguments you can pass to `rustdoc`:
```

Comment:
Right, you'd have to use RUSTDOCFLAGS for this, much like RUSTFLAGS for arguments to rustc.